configs/keycloak_bridge.yml

```diff
@@ -90,11 +90,11 @@ db-audit-ro-migration: false
 db-audit-ro-migration-version: 0.1
 db-audit-ro-connection-check: false
 
-# DB Configuration RW
+# DB Cloudtrust RW
 db-config-rw-host-port: 172.17.0.2:3306
 db-config-rw-username: bridge
 db-config-rw-password: bridge-password
-db-config-rw-database: configuration
+db-config-rw-database: cloudtrust
```

Comment:
Do we really want to rename this database? (already in production, referenced in multiple projects, ...)

Reply:
I have also some concerns about compatibility and I don't have the feeling the expresiveness of the content of the DB is improved with this new name

Reply:
Hello, 
I had the impression that the DB is actually called cloudtrust and not configuration. That's why I made these changes. 
But if want to keep the "configuration" name, it is fine by me.